bin/doctest

```diff
@@ -2,18 +2,25 @@
 
 'use strict';
 
-// Based on:
-// https://github.com/jashkenas/coffeescript/blob/1.9.1/src/command.coffee#L431-L441
-var args = process.argv.slice (1);
+var path = require ('path');
+
+var args = process.argv.slice (2);
 var idx = args.indexOf ('--nodejs');
-if (idx < 0 || idx === args.length - 1) {
-  require ('../lib/command');
-} else {
-  require ('child_process')
-  .spawn (process.execPath,
-          args[idx + 1]
-            .split (/\s+/)
-            .concat (args.slice (0, idx), args.slice (idx + 2)),
-          {cwd: process.cwd (), env: process.env, stdio: [0, 1, 2]})
-  .on ('exit', process.exit);
-}
+var esm = Number (process.versions.node.split ('.')[0]) >= 9;
+var flags = idx >= 0 && idx < args.length - 1;
```

Comment:
```suggestion
var flags = idx >= 0;
```

Reply:
`./bin/doctest -m esm test/esm/dependencies.mjs --nodejs` -> `TypeError: Cannot read property 'split' of undefined`

Reply:
The error message could be better, but it is an error, is it not?

```console
$ git commit --message
error: option `message' requires a value
```

Is there a reason to provide `--nodejs` without an associated value?

Reply:
There is no reason. What I'm trying to illustrate is that:

- *Before* applying your suggestion, associating no value with `--nodejs` gives us the following error message:
    ```txt
    error: option `    --nodejs <options>' argument missing
    ```
- *After* applying your suggestion, associating no value with `--nodejs` gives us the following error message:
    ```txt
    TypeError: Cannot read property 'split' of undefined
    ```

I consider that a regression.

Reply:
Oh, I see! I didn't realize we were already handling that error nicely. :)

Reply:
It's handled by Commander in `lib/program.js`. But in order for that to happen, we have to pass down the options in a specific way from `bin/doctest`.